shared/templates/template_OVAL_service_enabled

```diff
@@ -11,7 +13,7 @@
       <description>The {{{ SERVICENAME }}} service should be enabled if possible.</description>
     </metadata>
     <criteria comment="package {{{ PACKAGENAME }}} installed and service {{{ SERVICENAME }}} is configured to start" operator="AND">
-    <extend_definition comment="{{{ PACKAGENAME }}} installed" definition_ref="package_{{{ PACKAGENAME }}}_installed" />
+    <criterion comment="{{{ PACKAGENAME }}} installed" test_ref="{{{ package_installed_test_id }}}" />
```

Comment:
If a rule for the package installed already exists, this will end up creating two similar OVAL objects:
`test_service_fapolicyd_package_fapolicyd_installed`and
`test_package_fapolicyd_installed`

```
<ns8:rpminfo_object id="oval:ssg-obj_test_service_fapolicyd_package_fapolicyd_installed:obj:1" version="1">
    <ns8:name>fapolicyd</ns8:name>
</ns8:rpminfo_object>
```

``` 
<ns8:rpminfo_object id="oval:ssg-obj_test_package_fapolicyd_installed:obj:1" version="1">
    <ns8:name>fapolicyd</ns8:name>
</ns8:rpminfo_object>
```